profiles/api/oauth2.py

```diff
@@ -21,6 +24,7 @@
 from profiles.utilities import expires_at, remove_none_values
 
 LOGGER = logging.getLogger(__name__)
+PUBLISHER = get_publisher('profiles')
 
 
 def create_blueprint(orcid: Dict[str, str], clients: Clients, profiles: Profiles,
```

Comment:
I think we should inject the publisher as an argument to `create_blueprint` to avoid global state

Reply:
The `publisher` is now being injected as an argument into the `create_blueprint` function.